src/language-html/constants.evaluate.js

```diff
@@ -23,16 +23,23 @@ const CSS_DISPLAY_TAGS = {
   ...getCssStyleTags("display"),
 
   // TODO: send PR to upstream
-
   button: "inline-block",
 
   // special cases for some css display=none elements
   template: "inline",
   source: "block",
   track: "block",
   script: "block",
+  param: "block",
+  noscript: "none",
```

Comment:
Why `none`? How is it different from `block`?

Reply:
I  need  update tests, current example not good.

Reply:
`none` seems same as `inline`, I'll check if `noscript` can set to `block`

Reply:
On chrome, `<noscript>` is `inline`

![image](https://user-images.githubusercontent.com/172584/85858707-a6fe3b00-b7ee-11ea-98d4-fcca0a21a8d6.png)